package.json

```diff
@@ -11,9 +11,11 @@
   "license": "ISC",
   "dependencies": {
     "axios": "^0.19.0",
+    "bluebird": "^3.7.2",
```

Comment:
What's wrong with using the native Promise?

Reply:
There's a few patterns I couldn't use until installing bluebird.  They are:

`.map()` e.g. https://github.com/hmrc/x-govuk-component-renderer/pull/23/files#diff-bbe4898d5885bda2cc45415283c8c2e8R65-R69 and https://github.com/hmrc/x-govuk-component-renderer/pull/23/files#diff-4357639918214970bc2c9914ef1e4134R61-R66

`promisifyAll` e.g. https://github.com/hmrc/x-govuk-component-renderer/pull/23/files#diff-34a6d62af0cf0b784f8444529f3130efR7 which allows us to use https://github.com/hmrc/x-govuk-component-renderer/pull/23/files#diff-34a6d62af0cf0b784f8444529f3130efR74 and it will allow us to remove custom wrappers like this one https://github.com/hmrc/x-govuk-component-renderer/pull/23/files#diff-34a6d62af0cf0b784f8444529f3130efR13-R21

`.spread()` e.g. https://github.com/hmrc/x-govuk-component-renderer/pull/23/files#diff-4357639918214970bc2c9914ef1e4134R70-R73

If we could bump our node version we wouldn't need `promisifyAll` but I think `.spread()` and `.map()` still aren't supported natively.  I'm nervous to bump our node version above the version used by the libraries we're rendering - I think I'm just being a bit paranoid here.

Reply:
Fair enough :-)